server/api/game/controller.js

```diff
@@ -28,6 +29,9 @@ class Game {
       players: { [playerId]: {} },
       gridState: [],
       status: 0,
+      energizers: getEnergizers(),
```

Comment:
I think this should be done in method used for setting the initial game state.

Reply:
What difference does it make? Can you be more elaborate with your suggestion?